- remove enduser script with too generic names
- remove dependencies not required within ports

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -34,17 +34,7 @@ if flag in sys.argv:
 else:
     flag = False
 
-for filename in glob.glob('pwnlib/commandline/*'):
-    filename = os.path.basename(filename)
-    filename, ext = os.path.splitext(filename)
 
-    if ext != '.py' or '__init__' in filename:
-        continue
-
-    script = '%s=pwnlib.commandline.common:main' % filename
-    if not flag:
-        console_scripts.append(script)
-
 install_requires     = ['paramiko>=1.15.2',
                         'mako>=1.0.0',
                         'pyelftools>=0.2.4',
@@ -52,7 +42,6 @@ install_requires     = ['paramiko>=1.15.2',
                         'ropgadget>=5.3',
                         'pyserial>=2.7',
                         'requests>=2.0',
-                        'pip>=6.0.8',
                         'pygments>=2.0',
                         'pysocks',
                         'python-dateutil',
@@ -60,7 +49,7 @@ install_requires     = ['paramiko>=1.15.2',
                         'psutil>=3.3.0',
                         'intervaltree>=3.0',
                         'sortedcontainers',
-                        'unicorn>=1.0.2rc1,<1.0.2rc4', # see unicorn-engine/unicorn#1100, unicorn-engine/unicorn#1170, Gallopsled/pwntools#1538
+                        'unicorn',
                         'six>=1.12.0',
 ]
 
